.github/workflow-scripts/actOnLabel.js

```diff
@@ -102,11 +102,11 @@ module.exports = async (github, context, label) => {
       );
       await requestAuthorFeedback();
       return;
-    case 'Needs: Verify on Latest Version':
+    case 'Newer Patch Available':
```

Comment:
just a quick headsup, one this is merged go into the GH labels https://github.com/facebook/react-native/labels and rename the old/existing one too for consistency.

Reply:
Roger!